contrib/net/http/option.go

```diff
@@ -16,6 +16,9 @@ type config struct {
 // MuxOption represents an option that can be passed to NewServeMux.
```

Comment:
Can we add a comment here saying that this is now deprecated in favor of `Option`?